src/features/klant/_dev_fix_klanten.ts

```diff
@@ -0,0 +1,44 @@
+// ONLY use this during development, to fix existing test data without a subjectType
+import { fetchLoggedIn, throwIfNotOk, parseJson } from "@/services";
+import { KlantType } from "./types";
+
+const klantRootUrl = `${window.gatewayBaseUri}/api/klanten`;
+
+function getAllKlantenUrl() {
+  const url = new URL(klantRootUrl);
+  url.searchParams.set("limit", "999");
```

Comment:
It's more like a wildcard. But come to think of it, it might be nicer to turn this into a while loop of sorts, so we don't have too much requests at the same time. Thanks